agent/tool-scripts/haproxy-ocp

```diff
@@ -188,7 +188,7 @@ case "$mode" in
     start)
         mkdir -p $tool_output_dir
         pushd $tool_output_dir >/dev/null
-        echo "${BASH_SOURCE} --dir=${dir} --interval=${interval} --collect" >$tool_cmd_file
+        echo "${0} --dir=${dir} --interval=${interval} --collect" >$tool_cmd_file
```

Comment:
OK - whatever works and is robust enough is fine - I don't think there is much danger of sourcing this.